lib/ibm_watson/speech_to_text_v1.rb

```diff
@@ -50,6 +50,8 @@ module IBMWatson
   # The Speech to Text V1 service.
   class SpeechToTextV1 < IBMCloudSdkCore::BaseService
     include Concurrent::Async
+    DEFAULT_SERVICE_NAME = "speech_to_text"
```

Comment:
We decided to keep these changes out of the last release, so you'll need to rever these.

Reply:
done